### PR TITLE
Add Chromebook instructions

### PR DESCRIPTION
**Pull request by brownsarahm**
@ViableGolem4376 I found your contribution and opened this PR for your chromebook contribution, so we can iterate on it unitl it's ready to post.

### CODE REVIEW

File: syllabus/tools.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ Everything in this class will be tested with the up to date (or otherwise specif
 - Install python via [Anaconda](https://www.anaconda.com/products/individual)
 - if you use Windows, install Git with [GitBash](https://gitforwindows.org/) ([video instructions](https://youtu.be/339AEqk9c-8)).
 - if you use MacOS, install Git with the Xcode Command Line Tools. On Mavericks (10.9) or above you can do this by trying to run git from the Terminal the very first time.`git --version`
+- if you use chrome OS, refer to this pdf file: file:///home/chronos/u-1f0b530b8bb20c773cee66a2ceffc46503914ea3/MyFiles/Downloads/How%20to%20Install%20Anaconda%20Python%20onto%20a%20Chromebook.pdf
```
Comment (brownsarahm) [inline syllabus/tools.md:85]: This pdf isn't uploaded, it's a link to a local file on your computer. (the file:/// tells me that) 

Can you insert the actual text of the steps you followed?
```suggestion
- if you use chrome OS, follow the steps below:

    1. 
    1. 
```

**Comment by ViableGolem4376** [comment]
Ah I had a feeling that might happen, I will suggest a change with the
actual steps at some point soon.

**Comment by ViableGolem4376** [comment]
Ok I managed to get my terminal working, I just uninstalled it again and
redid the steps from before. It worked this time as I got a Jupyter
Notebook running. I have also added the list of instructions on the course
website.

**Comment by brownsarahm** [comment]
I don't see your updated instructions.  If you're reading this from e-mail, click the "view on github" link near the bottom and then click on "files changed" below the PR title "Add Chromebook instructions #5" and use the 3 dots menu to get to edit and then past your instructions

**Comment by brownsarahm** [comment]
closing because i found the changes you made on a new fork and opened #7

**Pull request closed without merging**